Readme.md

```diff
@@ -39,9 +41,10 @@ $ make install
 ## Usage
 
 ```console
-$ valkiria orchestrator ip=10.200.27.1 log=INFO
-$ valkiria agent ip=10.200.27.2 log=INFO
+$ valkiria orchestrator ip=10.200.27.1:9050 log=INFO
+$ valkiria agent ip=10.200.27.2:9050 log=INFO
 ```
 
 
-## Test environment
\ No newline at end of file
+## Warning
+Do not use in production, is in Alpha version. You need to run the agent process with root privileges. For tests of several packages is also necessary, else will be skipped.
```

Comment:
Not fully understand this sentence: "For tests of several packages is also necessary, else will be skipped."